### PR TITLE
Added component for new widget based dashboard

### PR DESCRIPTION
**Pull request by ivo-ritense**
https://ritense.tpondemand.com/entity/66300-befe-dashboard-tabs

### CODE REVIEW

File: projects/valtimo/dashboard/src/lib/components/widget-dashboard/widget-dashboard.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div *ngIf="dashboards$ | async as dashboards">
+  <div *ngIf="dashboards.length == 1">
+    <h4>{{dashboards[0].title}}</h4>
+    <valtimo-widget-dashboard-content [dashboard]="dashboards[0]"></valtimo-widget-dashboard-content>
```
Comment (teodor-ritense) [inline projects/valtimo/dashboard/src/lib/components/widget-dashboard/widget-dashboard.component.html]: Add a newline here

File: projects/valtimo/dashboard/src/lib/components/widget-dashboard/widget-dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  </div>
+  <div *ngIf="dashboards.length > 1">
```
Comment (teodor-ritense) [inline projects/valtimo/dashboard/src/lib/components/widget-dashboard/widget-dashboard.component.html:26]: Add a newline here

File: projects/valtimo/dashboard/src/lib/components/widget-dashboard/widget-dashboard.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import {Component} from '@angular/core';
+import {DashboardService} from '../../services';
+
+@Component({
+  selector: 'valtimo-widget-dashboard',
+  templateUrl: './widget-dashboard.component.html',
+  styleUrls: ['./widget-dashboard.component.css'],
+})
+export class WidgetDashboardComponent {
+  public dashboards$ = this.dashboardService.getDashboards()
```
Comment (teodor-ritense) [inline projects/valtimo/dashboard/src/lib/components/widget-dashboard/widget-dashboard.component.ts]: Please specify a type for dashboards$

File: projects/valtimo/dashboard/src/lib/services/dashboard.service.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {ConfigService} from '@valtimo/config';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DashboardService {
+  private valtimoApiConfig: any;
+
+  constructor(private http: HttpClient, private configService: ConfigService) {
```
Comment (teodor-ritense) [inline projects/valtimo/dashboard/src/lib/services/dashboard.service.ts]: Add readonly

File: projects/valtimo/dashboard/src/lib/services/dashboard.service.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    this.valtimoApiConfig = configService.config.valtimoApi;
```
Comment (teodor-ritense) [inline projects/valtimo/dashboard/src/lib/services/dashboard.service.ts]:
```suggestion
    this.valtimoApiConfig = configService.config.valtimoApi.endpointUri;
```

File: projects/valtimo/dashboard/src/lib/services/dashboard.service.ts [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import {Observable} from 'rxjs';
+import {Dashboard} from '../models';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {ConfigService} from '@valtimo/config';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DashboardService {
+  private valtimoApiConfig: any;
```
Comment (teodor-ritense) [inline projects/valtimo/dashboard/src/lib/services/dashboard.service.ts]:
```suggestion
  private valtimoApiConfig: string;
```

**Review by teodor-ritense — DISMISSED** [review]

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**